components/Budget/DeleteExpenseModal.jsx

```diff
@@ -0,0 +1,12 @@
+import { View } from "react-native";
+import Subtitle from "../TextStyles/Subtitle";
+import DeleteButton from "./DeleteButton";
+
+export default function DeleteExpenseModal({ onPress }) {
+    return (
+        <View className="flex-col items-center mb-8">
+            <Subtitle extraClassName={"text-center"}>Are you sure you want to delete this expense?</Subtitle>
+            <DeleteButton onPress={onPress} />
```

Comment:
I like the delete button on the main page but not totally sure about it on the modal

Reply:
have changed the padding, let me know what u think